charts/gardener/controlplane/charts/runtime/templates/apiserver/configmap-admission-config.yaml

```diff
@@ -1,4 +1,6 @@
-{{- if and .Values.global.apiserver.enabled .Values.global.apiserver.admissionConfig }}
+{{- with .Values.global.apiserver }}
+{{- if and .enabled (or .admission.validatingWebhook.kubeconfig .admission.mutatingWebhook.kubeconfig .admission.plugins) }}
```

Comment:
Can you use a template function, i.e. `admissionConfigEnabled` and re-use it everywhere you have these OR conditions?